images/db/migrate/20150430171341_translate_refinery_images.rb

```diff
@@ -0,0 +1,14 @@
+class TranslateRefineryImages < ActiveRecord::Migration
+  def self.up
+    Refinery::Image.create_translation_table!({
+      image_alt: :string,
+      image_title: :string
+    }, {
+      :migrate_data => true
+    })
+  end
+
+  def self.down
+    Refinery::Image.drop_translation_table! migrate_data: true
```

Comment:
I hate to say it but this is going to suffer the same problems as #2715 - can you please apply the same solution as in 8f56bf47a7ecd6d348b26d9531ff5e4eb03d5f68 ?